lib/API/Startup.js

```diff
@@ -59,10 +59,11 @@ module.exports = function(CLI) {
     return hash_map[init_systems[i]];
   }
 
-  CLI.prototype.uninstallStartup = function(platform, cb) {
+  CLI.prototype.uninstallStartup = function(platform, opts, cb) {
     var commands;
     var that = this;
     var actual_platform = detectInitSystem();
+    var service_name = 'pm2-' + (opts.user || process.env.USER);
```

Comment:
Yeah this one is a bit interesting.. 
Maybe do a check if multiple pm2 services are present when running `pm2 unstartup` and then writing

> There are other pm2 services present for users x, y, z. Run `pm2 unstartup $USER` if you want to remove them as well